src/fetchtastic/download/client_app.py

```
@@ -0,0 +1,1029 @@
+                error_type=error_type,
+            )
+
+    def download_apk(self, release: Release, asset: Asset) -> DownloadResult:
+        """Compatibility alias."""
+        return self.download_app(release, asset)
+
+    def download_desktop(self, release: Release, asset: Asset) -> DownloadResult:
+        """Compatibility alias."""
+        return self.download_app(release, asset)
+
+    def is_release_complete(self, release: Release) -> bool:
```

Ruff (0.15.12)

[warning] Too many return statements (8 > 6)

(PLR0911)